day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,26 @@
+# FizzBuzz
+def fizzbuzz (low, high)
+  while low < high
+    if low % 15 === 0
+      print "FizzBuzz, "
+    elsif low % 5 === 0
+      print "Buzz, "
+    elsif low % 3 === 0
+      print "Fizz, "
+    else
+      print "#{low}, "
+    end
+    low += 1
+  end
+  if high % 15 === 0
+    print "FizzBuzz"
+  elsif high % 5 === 0
+    print "Buzz"
+  elsif high % 3 === 0
+    print "Fizz"
```

Comment:
Seems like these lines are repeated from inside your `while` loop. This is a good candidate to refactor so you don't have repeating conditionals with the same output.